web-console/developer/topology/topology-serverless/topology-serverless.md

```diff
@@ -0,0 +1,39 @@
+# Serverless
+
+## Event Sources
+In 4.3, the console will visualize a subset of the event source types in the topology view.  This list includes:
+    KafkaSource
+    CamelSource
+    CronJobSource
+    ContainerSource
+    ApiServerSource
+The documentation about the event source types that are supported can be found [here](https://knative.dev/docs/eventing/)
+
+Event Sources have a sink connector to the Knative service.
+
+![event source top](img/EventSource.png)
+
+When the event source is selected, the side panel is displayed on the right side.
+
+The side panel displays the Overview of the selected Event Source.
+![event source overview](img/EventSource-Overview.png)
```

Comment:
@mceledonia - Have you guys covered the "selected" treatment in the topology designs? It could be just that it's a flat image, but I wonder if the treatment should stronger to be clear which item the details panel is pertaining to.

Reply:
@lizsurette the selected treatment for event sources and regular workloads is the same (blue outer edge).  It's recently been implemented, so we can definitely get some feedback on that!

Reply:
@serenamarie125 sounds great!

Reply:
+1, also this is looking good to me!